tests/foreman/cli/test_discoveryrule.py

```diff
@@ -424,13 +558,21 @@ def test_positive_view_existing_rule_with_non_admin_user(self):
 
         @Assert: Rule should be visible to non-admin user.
 
-        @caseautomation: notautomated
-
         @CaseLevel: Integration
         """
+        rule = make_discoveryrule({
+            u'enabled': 'false',
+            u'search': "last_report = Today",
+            u'organization-ids': self.org['id'],
+            u'location-ids': self.loc['id'],
+            u'hostgroup-id': self.hostgroup['id'],
+        })
+        rule = DiscoveryRule.with_user(
+            self.user_reader['login'],
+            self.user_reader['password']
+        ).info({u'id': rule['id']})
```

Comment:
Please add any assertion here, for example assert that returned discovery rule name is the same as it was during creation.
Without any assertion you're basically verifying zero return_code, but no guarantee anything was actually returned.